src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -135,7 +135,9 @@ public class UTIL_ACCT_Naming {
         }
 
         if (defaultRecTypeId == hhAccountRecordTypeId) {
-            dynamicSoql += 'Phone, Fax FROM Contacts WHERE Exclude_from_Household_Name__c != true)';
+            dynamicSoql += 'Phone, Fax FROM Contacts WHERE Exclude_from_Household_Name__c != true ORDER BY LastName, FirstName)';
```

Comment:
@kdy618 So, unfortunately, the Contact Name is an encryptable field and, thus, can't be used in the ORDER BY clause :-( (See: https://help.salesforce.com/articleView?id=security_pe_considerations_general.htm&type=5)

The fields can be queried though, so we might need to do the ordering ourselves...

If you run the "setup_encryption" flow in your dev scratch org, you should then be able to go to the Platform Encryption | Encryption Policy settings and encrypt the Contact Name field. I'm assuming that after you do so, you will get errors here since the code is ordering by LastName & FirstName. (The errors will either be instant or may not appear until trying to re-deploy the code or run the unit tests.)

Does this all sound right, @lifewithryan?

Reply:
I haven't read through the main refactoring PR yet, but it would be a good idea to encrypt all of the fields on the Contact to try to make sure there aren't other areas that we may need to fix in the refactoring as well.

Reply:
^ Definitely test this first! I might be wrong and I would hate to make you go through the work of fixing this for no reason.

Reply:
@robertbwatson , good point and catch. I'll work on testing out the encryption to check if it breaks.